.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Description
+
+Please explain the changes you made here.
+Explain the **motivation** for making this change. What existing problem does the pull request solve?
+
+### Checklist
+
+- [ ] Code compiles correctly
```

Comment:
We should have a check box for 
`New tests covering changes and new functionality`